### DISCUSSION TITLE
REDACTED FOR PRIVACY

### DISCUSSION THREAD

**Discussion by ghost** [rbenv/ruby-build#2195] [Build failures]
REDACTED FOR PRIVACY

**Comment by MichalDobry** [comment]
i have got exactly the same issue on M2, Ventura 13.3.1
with `rbenv install 3.1.2`

**Comment by MichalDobry** [comment]
ok, i resolved this by using chruby:
[chruby install](https://www.moncefbelyamani.com/how-to-install-xcode-homebrew-git-rvm-ruby-on-mac/)
then i set local version for my project by:
`rbenv local 3.1.4`

**Comment by mislav** [comment]
@casrass If compiling OpenSSL fails, you can try to avoid compiling OpenSSL altogether by installing it via Homebrew and linking Ruby to that: https://github.com/rbenv/ruby-build/wiki#macos

Note that the full build log is in the file referenced in your output and that only the last 10 lines of the log are printed in what you shared here. If you wanted to share your build failure for others to try to guess what happened here, my suggestion would be to paste the whole build log as a Gist. But, my primary course of action would be to avoid building OpenSSL by providing a pre-installed one for Ruby.

**Reply by false9striker** [reply]
But installing openssl@1.1 itself failes in M1 Mac with 13.3.1 (a)

**Reply by mislav** [reply]
@false9striker If `brew install openssl@1.1` or `brew install openssl@3` fails, that's either a Homebrew problem or the problem of the incomplete or messed up build system on the machine in question, not a ruby-build problem. Note that most of the time, Homebrew just downloads and extracts precompiled binaries (that it calls "bottles") for your architecture, instead of actually compiling software. If you can reproduce the problem reliably, have run `brew doctor`, and have logs to back your findings, you could try reporting this to `homebrew-core`.

I have an M1 Mac running 13.3.1 and `brew install openssl` (any version) had worked for me (and most ruby-build users) for years already.
